mem_sql/src/main/resources/log4j2.properties

```diff
@@ -12,13 +12,16 @@
 # If not, see <http://www.gnu.org/licenses/>.
 #
 rootLogger.level = info
+rootLogger.includeLocation = true
 rootLogger.appenderRef.file.ref = File1
 
-appender.file.type = File
+appender.file.type = RandomAccessFile
```

Comment:
what does this do?

Reply:
RandomAccessFile is buffered and faster than a normal file appender, even for synchronous loggers.  I think I read somewhere they are needed for full asynchronous logging support, but I can't find that article now.  
At the following link, search for "FileAppender vs. RandomAccessFileAppender" for a performance comparison using synchronous loggers:
https://logging.apache.org/log4j/log4j-2.4/manual/async.html